pkg/controller/nodenetworkconfigurationpolicy/policyconditions/conditions.go

```diff
@@ -85,6 +85,31 @@ func setPolicyFailedToConfigure(conditions *nmstatev1alpha1.ConditionList, messa
 	)
 }
 
+func nodesRunningNmstate(cli client.Client) ([]corev1.Node, error) {
+	nodes := corev1.NodeList{}
+	err := cli.List(context.TODO(), &nodes)
+	if err != nil {
+		return []corev1.Node{}, errors.Wrap(err, "getting nodes failed")
+	}
+
+	pods := corev1.PodList{}
+	err = cli.List(context.TODO(), &pods) // TODO filter nmstate pods
```

Comment:
What about the todo? Once we add an operator, we would get masters listed here which may not have handlers running.

Reply:
I forgot the filtering... Let's add a unit test for that.

Reply:
Ok addind filter on `app: kubernetes-nmstate` and unit test, also operator should have diffrent app.